client/src/components/Collections/wizard/SourceFromCollection.vue

```diff
@@ -0,0 +1,29 @@
+<script setup lang="ts">
+import { BCard, BCardTitle } from "bootstrap-vue";
+
+import { borderVariant } from "@/components/Common/Wizard/utils";
+
+interface Props {
+    selected: boolean;
+}
+
+defineProps<Props>();
+
+const emit = defineEmits(["select"]);
+</script>
+
+<template>
+    <BCard
+        data-import-source-from="collection"
+        class="wizard-selection-card"
+        :border-variant="borderVariant(selected)"
+        @click="emit('select', 'collection')">
+        <BCardTitle>
+            <b>An Existing Collection</b>
+        </BCardTitle>
+        <div>
+            <!-- TODO: replace collection with an english expression -->
+            Fill in sample sheet data from an existing collection.
```

Comment:
I do like this, all of the other options should be much shorter with a help button that has more details.

Reply:
Is that the right option or should we just have two rows of buttons? I feel like my text descriptions are not longer than the ones @davelopez has been using?